packages/ckb-sdk-core/src/index.ts

```diff
@@ -89,34 +89,28 @@ class Core {
   }
 
   public loadSecp256k1Dep = async () => {
-    /**
-     * cell list
-     * @link https://github.com/nervosnetwork/ckb/blob/dbadf484cea6bdba0329d58102726068be997a50/docs/hashes.toml
-     */
     const block = await this.rpc.getBlockByNumber('0x0')
-    if (!block) throw new Error('Cannot load the genesis block')
-    const secp256k1CodeTx = block.transactions[0]
-    if (!secp256k1CodeTx) throw new Error('Cannot load the transaction which has the secp256k1 code cell')
-    if (!secp256k1CodeTx.outputs[1]) {
-      throw new Error('Cannot load the secp256k1 cell because the specific output is not found')
-    }
-    if (!secp256k1CodeTx.outputs[1].type) {
-      throw new Error('Secp256k1 type script not found in the cell')
-    }
 
-    const secp256k1TypeHash = this.utils.scriptToHash(secp256k1CodeTx.outputs[1].type)
+    /* eslint-disable */
```

Comment:
Why add `eslint-disable` ?

Reply:
> Why add `eslint-disable` ?

eslint-prettier is working on it. I’ll remove this comment once it’s published.

Reply:
OK. Adding `TODO` maybe more better.